sky/resources.py

```diff
@@ -162,6 +162,8 @@ def __init__(
                 }
         self._is_image_managed = _is_image_managed
 
+        if isinstance(disk_tier, str):
+            disk_tier = resources_utils.DiskTier[str(disk_tier).upper()]
```

Comment:
Should this be the following?
```suggestion
            disk_tier = resources_utils.DiskTier(str(disk_tier).lower())
```

Reply:
No, the `enum.Enum` actually stores the variable name for the key, not the value...

A minimal test script:

```python
# test_enum.py
import enum

class E(enum.Enum):
    a = 'a'
    B = 'b'
    c = 'C'
print(E['a'])
print(E['a'].value)
print(E['B'])
print(E['B'].value)
print(E['c'])
print(E['c'].value)
```

```bash
✗ py test_enum.py
E.a
a
E.B
b
E.c
C
```

Reply:
Oh, I mean using `__call__`

Reply:
Oh that makes sense! I don't know this syntax before. Thanks!